ambry-api/src/test/java/com.github.ambry/rest/RestUtilsTest.java

```diff
@@ -179,7 +179,6 @@ public void getUserMetadataGoodInputTest()
     userMetadata.put(RestUtils.Headers.USER_META_DATA_HEADER_PREFIX + "key1", "value1");
     userMetadata.put(RestUtils.Headers.USER_META_DATA_HEADER_PREFIX + "key2", "value2");
     setUserMetadataHeaders(headers, userMetadata);
-    verifyBlobPropertiesConstructionSuccess(headers);
```

Comment:
Removed because the test isn't for verifying blob properties (it is for user metadata).